signalfx-tracing/signalfx-python-tracing/django/blog/blog/settings.py

```diff
@@ -0,0 +1,139 @@
+# IMPORTANT:
+# The only requirement for auto-instrumentation is to add
+# 'signalfx_tracing' to installed apps.
```

Comment:
nit: this is not entirely accurate since the tracer isn't created by this process (you still have to use sfx-py-trace or specify the tracer callable.  I'd say something like "To enable auto-instrumentation of Django apps, you must add the..."

Reply:
Correct, thanks. a bunch! Updated!